select_assignment2_flicks/FilterViewController.swift

```diff
@@ -0,0 +1,175 @@
+//
+//  FilterViewController.swift
+//  select_assignment2_flicks
+//
+//  Created by Hoa Tran on 7/10/16.
+//  Copyright © 2016 com.noron. All rights reserved.
+//
+
+import UIKit
+import SnapKit
+
+//MARK: -Constants
+extension FilterViewController {
+    @nonobjc static let CONTENT_CONTROL_INDEX = 0
```

Comment:
I had a run-time error (definition must be dynamic or sth...) in MovieCell when I tried to move the constants inside that class, so I googled I find the solution to add @nonobjc but it didn't work at all, so I haven't removed this one, and to be honest, I don't understand what this annotation means. :(